janitor/functions/dropnotnull.py

```diff
@@ -8,16 +8,22 @@
 @pf.register_dataframe_method
 @deprecated_alias(column="column_name")
 def dropnotnull(df: pd.DataFrame, column_name: Hashable) -> pd.DataFrame:
-    """
-    Drop rows that do not have null values in the given column.
+    """Drop rows that do *not* have null values in the given column.
 
     This method does not mutate the original DataFrame.
 
-    Example usage:
+    Example:
 
-    ```python
-        df = pd.DataFrame(...).dropnotnull('column3')
-    ```
+        >>> import numpy as np
+        >>> import pandas as pd
+        >>> import janitor
+        >>> df = pd.DataFrame({"a": [1., np.NaN, 3.], "b": [None, "y", "z"]})
```

Comment:
Not a bad idea to show the original dataframe as well. What do u think?

Reply:
Yes, it's a good idea. The challenge then shifts to making a minimal working example that isn't too verbose.